src/config/session-management.md

```diff
@@ -41,6 +41,14 @@ There is an alternative D-Bus configuration which takes advantage of elogind for
 features such as seat detection. It requires installing the `dbus-elogind`,
 `dbus-elogind-libs` and `dbus-elogind-x11` packages.
 
+If you will use Sway, you should have installed these packages in your system to work perfectly:
+- `elogind`
+- `polkitd-elogind`
+- `dbus-elogind-libs`
+- `dbus-elogind-x11`
+- `dbus-elogind`
```

Comment:
All of this is already mentioned in https://docs.voidlinux.org/config/session-management.html?highlight=elogind#elogind (save for the polkit part, see #694). Also, you can run Sway with seatd just fine, as @Duncaen states.

Reply:
also, the `dbus-elogind-*` packages are probably not necessary, regular dbus should be fine